pkg/cmd/attestation/verify/policy.go

```diff
@@ -51,7 +49,8 @@ func buildCertificateIdentityOption(opts *Options, runnerEnv string) (verify.Pol
 		return nil, err
 	}
 
-	issuerMatcher, err := verify.NewIssuerMatcher(opts.OIDCIssuer, "")
+	// Accept any issuer, we will verify the issuer as part of the extension verification
+	issuerMatcher, err := verify.NewIssuerMatcher("", ".*")
```

Comment:
Currently, the issuer verification is handled by the `sigstore-go` verifier . . . however, the existing logic doesn't allow us to properly detect the customized issuer string and alter the error message accordingly. 

Instead, we're going to let the verifier accept ANY issuer value and perform the issuer comparison as part of the extension verification done within the `verify` command itself.